packages/web-runtime/src/index.ts

```diff
@@ -74,6 +79,24 @@ export const bootstrapApp = async (configurationPath: string): Promise<void> =>
     translations: merge(translations, customTranslations)
   })
   announceClientService({ app, runtimeConfiguration, configurationManager, store })
+
+  // TODO: move to announceArchiverService function
```

Comment:
ah, your comment about followup was not visible in my review anymore (probably because of history change). fine by me if you do that in a followup. but it's easily forgotten and not a hard task, so please better earlier than later ;-)